repair_tests/preview_repair_test.py

```diff
@@ -80,9 +83,18 @@ def test_preview(self):
         # ...and everything should be in sync
         result = node1.repair(options=['ks', '--preview'])
         assert "Previewed data was in sync" in result.stdout
+        # data is repaird, previewFailure metric should remain same
+        assert preivew_failure_count(node1) == 2
 
         result = node1.repair(options=['ks', '--preview', '--full'])
         assert "Previewed data was in sync" in result.stdout
+        assert preivew_failure_count(node1) == 2
 
         result = node1.repair(options=['ks', '--validate'])
         assert "Repaired data is in sync" in result.stdout
+
+
+def preivew_failure_count(node):
+    preview_failure_size_mbean = make_mbean('metrics', type='Repair', name='PreviewFailures')
```

Comment:
Nit: we could name the local variable just `mbean` instead of `preview_failure_size_mbean `, feel free to ignore if you don't agree.